src/UI/Component/Panel/Factory.php

```diff
@@ -95,4 +95,33 @@ public function sub($title,$content);
 	 */
 	public function report($title,$sub_panels);
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *       Listing Panels are used to list items following all one
```

Comment:
@alex40724 Good point. I updated the information about this in a comment bellow.